### ISSUE TITLE
Ignoring line: RequestListener : $blank$ <com.android.volley.Response.Listener<T>>

### ISSUE THREAD

**Issue by Ligator** [HipByte/RubyMotionSamples#64]
Trying to get running the Android example: [Weather](https://github.com/HipByte/RubyMotionSamples/tree/master/android/Weather). 
I am getting this error every time:

```
$ rake device
...
Create ./build/Development-19/lib/armeabi/libpayload.so
Ignoring line: RequestListener : $blank$ <com.android.volley.Response.Listener<T>>
Method declaration outside class definition
```

The error is about the file `vendor/volley-1.0.9.jar`, but I can't solve it. I already tried with the [VolleyMotion gem](https://github.com/TigerWolf/volley_motion) and with the latest volley version from [Maven Central Repository] (http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.mcxiaoke.volley%22).

What am I doing wrong?
Help me, please!

I am using API level 19.
RubyMotion 3.1

**Comment by GantMan** [comment]
How are other projects?   Do they work?

Have you ran the bluepotion project?  https://github.com/infinitered/bluepotion

I'm interested if those work.

**Comment by Ligator** [comment]
Hi @GantMan,
I tried to run all examples. I had troubles with these examples:

* **AsyncTaskExample** is not supported by my RM version (It requires at least RM 3.7).
* **Weather** doesn't compile. The error is shown in my first comment.
* **Conference** doesn't open, it is raising an exception.
* **ServiceDemo** is crashing when I push the 'Start' button (but the music is actually played).

But I just want be able to make a JSON call. :(

About BluePotion, I am excited to know that the project is very active.
I already have used RedPotion for iOS, but I don't know where I can find the documentation for BluePotion. Something like this http://redpotion.readthedocs.org/en/latest/

Is BluePotion including networking libraries out of the box?

API level 19. RM 3.1

**Comment by GantMan** [comment]
So, BluePotion is still alpha, and really undocumented.   It grows every day!   We really need to lock down testing for it and docs, but code wins over all right now.

BluePotion does come with a library.  It will also come with a lil bit of undocumented headache :+1:

**Comment by Ligator** [comment]
Today I found the Volley wrapper for BluePotion, it looks awesome!.
And yes, the documentation is commented into the code, it is enough.

Unfortunately, I was not able to run the BluePotion template, I got an error already reported and fixed in recent RM versions.

I think my trouble is not about libraries, but it's about my RM version. 
I'll renew my license. If it doesn't work, I'll be back to this issue.

Thank you @GantMan for your support. :)
